src/clarity-angular/modal/modal.ts

```diff
@@ -106,8 +107,16 @@ export class Modal implements OnChanges, OnDestroy {
     ngOnChanges(changes: {[propName: string]: SimpleChange}): void {
         if (!this.bypassScrollService && changes && changes.hasOwnProperty("_open")) {
             if (changes._open.currentValue) {
+                const hostElement = this.elementRef.nativeElement;
```

Comment:
Instead of getting the `ElementRef`, can we not just use a class which is applied to the host when the modal is not open? The class just has `display: none` on it and is removed when the modal is open.

Reply:
I have removed `ElementRef ` and used  `@HostBinding("class.open")` so open class is added and added `:host { display: none; }  :host.open { display: inline; }` css.